docs/guides/faq/index.html

```diff
@@ -13,7 +13,7 @@ <h2 id="what-are-web-components">What are Web Components?</h2>
 
     <br />
 
-    <dl class="hxList metadata hxBox-sm">
+    <dl class="hxList metadata hxBox hxSm">
```

Comment:
I think we can get away with normalizing lists in this file to `<dl class="hxList metadata">` (use default padding instead of small).